app/src/main/kotlin/com/wire/android/ui/main/message/Message.kt

```diff
@@ -0,0 +1,340 @@
+package com.wire.android.ui.main.message
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.wire.android.R
+import com.wire.android.ui.common.LegalHoldIndicator
+import com.wire.android.ui.common.MembershipQualifierLabel
+import com.wire.android.ui.common.UserProfileAvatar
+import com.wire.android.ui.main.conversation.model.AvailabilityStatus
+import com.wire.android.ui.main.conversation.model.Membership
+import com.wire.android.ui.theme.body02
+import com.wire.android.ui.theme.wireColorScheme
+
+@Composable
+fun Message(
```

Comment:
Let's try to be a bit more specific here, otherwise we might get confused with possible model objects. Shall we rename to 
```suggestion
fun MessageItem(
```